### PR TITLE
Create file, first lesson

### PR DESCRIPTION
**Pull request by Resires**
Task list:

Snakify / Lesson 1 / Problems (all 6)
https://snakify.org/lessons/print_input_numbers/problems/

Snakify / Lesson 2 / Problems (first 9, skip "Clock face - 2")
https://snakify.org/lessons/integer_float_numbers/problems/

### CODE REVIEW

File: students/uss_tomasz/lesson_01_basics/nowy.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+later_seconds = 3600 * later_h + 60 * later_m + later_s
+earlier_seconds = 3600 * earlier_h + 60 * earlier_m + earlier_s
+print("Seconds difference = ", later_seconds - earlier_seconds)
+
+# Problem School desks
+a = int(input("Students in first class = "))
+b = int(input("Students in second class = "))
+c = int(input("Students in third class = "))
+print("Desks needed = ", ceil(a / 2) + ceil(b / 2) + ceil(c / 2))
```
Comment (jedzej) [inline students/uss_tomasz/lesson_01_basics/nowy.py]: it's fine, you could also use list for better scalability:
```
students_counts = [
  int(input()),
  int(input()),
  int(input()),
]
sum([ceil(count / 2) for count in students_counts])
```

File: students/uss_tomasz/lesson_01_basics/nowy.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+variable = int(input("Variable = "))
+print("Last digit of ", variable, "is", variable % 10)
+
+# Problem Tens digit
+variable = int(input("Variable = "))
+print("Digit of tenths of ", variable, "is", variable % 100 // 10)
+
+# Problem Sum of digits
+variable = int(input("Variable made of three digits = "))
+sum = variable % 10
```
Comment (jedzej) [inline students/uss_tomasz/lesson_01_basics/nowy.py]: sum is a built-in function, rename this variable not to shadow the built-in
```
> sum
<built-in function sum>
```

File: students/uss_tomasz/lesson_01_basics/nowy.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+number_3 = int(input("Third number = "))
+
+print("Sum of them is", number_1 + number_2 + number_3)
+
+# Problem Hi John
+name = input("What is your name?")
+print("Hi", name)
+
+# Problem Square
+variable = int(input("Input variable = "))
```
Comment (jedzej) [inline students/uss_tomasz/lesson_01_basics/nowy.py]: find a meaningful names for the variables

File: students/uss_tomasz/lesson_01_basics/nowy.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+print("Squared = ", variable ** 2)
+
+# Problem Area of right-angled triangle
+baseLenght = float(input("Length of triangle base = "))
```
Comment (jedzej) [inline students/uss_tomasz/lesson_01_basics/nowy.py]: read PEP8 guidelines regarding variable names and adapt the code accordingly
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

**Review by jedzej — CHANGES_REQUESTED** [review]

**Review by jedzej — APPROVED** [review]
keep up good work 🔥

**Comment by Resires** [comment]
Modification finished

**Pull request merged**